jsonschema-hyperschema.xml

```diff
@@ -442,8 +459,17 @@
 
         <section title="Link Description Object">
             <t>
-                A Link Description Object (LDO) is used to describe a single link relation from the
-                instance to another resource.
+                A Link Description Object (LDO) is a serialization of the abstract model specified
+                in Section 2 of <xref target="I-D.nottingham-rfc5988bis"/>, defines a link as
```

Comment:
Yeah, I meant to have a "which" between the xref/comma and "defines a link", will fix, thanks!